packages/backend/src/users/users.controller.ts

```diff
@@ -30,6 +30,11 @@ export class UsersController {
     return await this.userStoreService.findOne(id);
   }
 
+  @Patch(':id')
+  async update(@Param('id') id: string, @Body() updateUserDto: UpdateUserDto) {
+    return await this.userStoreService.update(id, updateUserDto);
```

Comment:
just remove the await and then you are good to merge!

Reply:
once you have gone through the PR checklist at least

Reply:
if there's no await this would return a promise. i forgot to exec this, should be fine in latest commit with the await

Reply:
Its an async function it is supposed to return a promise. Nestjs handles it correctly

Reply:
oh right what am i thinking

Reply:
There should not be a difference between return await vs return